pkg/solana/transmitter.go

```diff
@@ -84,21 +84,10 @@ func (c *ContractTracker) Transmit(
 
 	// Send transaction, and wait for confirmation:
 	go func() {
-		ctx, cancel := context.WithTimeout(context.Background(), c.client.txTimeout)
-		defer cancel()
-		txSig, err := c.client.rpc.SendTransactionWithOpts(
-			ctx, // does not use libocr transmit context
-			tx,
-			c.client.skipPreflight,
-			c.client.commitment,
-		)
-
-		if err != nil {
-			c.lggr.Errorf("error on Transmit.SendAndConfirmTransaction: %s", err.Error())
-			return
+		// send to tx manager
+		if err := c.txManager.Enqueue(c.StateID.String(), tx); err != nil {
+			c.lggr.Errorf("error on Transmit.txManager.Enqueue: %s", err.Error())
```

Comment:
Since `Enqueue` is a non-blocking channel send attempt, could this be called syncronously instead of from a spawned goroutine?